src/main/java/com/google/devtools/build/lib/starlarkbuildapi/proto/ProtoBootstrap.java

```diff
@@ -28,23 +28,23 @@
   /** The name of the proto info provider in Starlark. */
   public static final String PROTO_INFO_STARLARK_NAME = "ProtoInfo";
 
-  /** The name of the proto toolchain info provider in Starlark. */
-  public static final String PROTO_TOOLCHAIN_INFO_STARLARK_NAME = "ProtoToolchainInfo";
-
   /** The name of the proto namespace in Starlark. */
   public static final String PROTO_COMMON_NAME = "proto_common";
 
   private final ProtoInfoProviderApi protoInfoApiProvider;
+  private final ProtoToolchainInfoApi.Provider protoToolchainInfoApi;
   private final Object protoCommon;
   private final StarlarkAspectApi protoRegistryAspect;
   private final ProviderApi protoRegistryProvider;
 
   public ProtoBootstrap(
       ProtoInfoProviderApi protoInfoApiProvider,
+      ProtoToolchainInfoApi.Provider protoToolchainInfoApi,
```

Comment:
ProtoInfo is passing the actual Api class, not the Api.Provider. Why the difference here?

Reply:
`ProtoInfo` also passes the provider, but they named it `ProtoInfoApi.ProtoInfoProviderApi`

Reply:
Ah, I see. Thanks for clarifying.